README.md

```diff
@@ -25,12 +25,81 @@ Quality assurance framework for Magnetic Resonance Imaging
 ## Overview
 
 ---
+hazen is a software framework for performing automated analysis of MRI Quality Assurance data.
 
+Some examples of what hazen can perform:
 
+| SNR measurements   | Spatial Resolution measurements |
+| ------------------ | ------------------------------- |
+| _insert snr image_ | _insert spatial res image_      |
+| snr explanation    | spatial res explanation         |
+
+
+## Installation
+
+---
+
+### Prerequisites
+
+ - Python v3.9
+   - conda is required for hazen installation on Mac M1 silicon (arm64 architecture)
+ - Git
+
+### Install
+
+First, clone this repo, then follow the instructions for your operating system. To clone:
+```bash
+git clone git@github.com:GSTT-CSC/hazen.git
+```
+
+#### MacOS
+
+```bash
+# Download and install conda (we have tested with both Miniforge and Miniconda):
+brew install miniforge
```

Comment:
Can you clarify what non-m1 macs need to please.

Reply:
Yeah.

@laurencejackson – can you do a test install on your Intel Macbook? Using the requirements.txt.